pages/index.vue

```diff
@@ -146,25 +146,6 @@ defineOgImage({
                 />
                 Join our community on Slack
               </AppButton>
-
```

Comment:
No one has ever messaged up that the link hasn't worked and this is the most ugly warning so i am just deleting it lol, people can contact us if we they have issues :P